onnxruntime/core/providers/cpu/math/quantize_linear_matmul.cc

```diff
@@ -53,9 +64,9 @@ Status QLinearMatMul::Compute(OpKernelContext* ctx) const {
               "QLinearMatmul : result zero point must be a scalar or 1D tensor of size 1");
 
   // validate scale
-  const auto* a_scale = ctx->Input<Tensor>(1);
-  const auto* b_scale = ctx->Input<Tensor>(4);
-  const auto* y_scale = ctx->Input<Tensor>(6);
+  const auto* a_scale = ctx->Input<Tensor>(IN_Ascale);
```

Comment:
why does this use In_Ascale while the other classes use InIdx::Ascale

Reply:
changing all occurrences to be consistent  with  existing ones in dml